Gemfile

```diff
@@ -40,13 +41,15 @@ group :development, :test do
   gem 'rubocop', '~> 1.12'
   gem 'rubocop-rails', '~> 2.9'
   gem 'rubocop-rspec'
-  gem 'simplecov'
-  gem 'simplecov-json'
```

Comment:
moved gems only used for testing to the "test" group (so they only are installed on test environemnts)